tools/hyperref-check.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Extracting all broken hyperref detected by PDFTex
+for file in "./main/acle.md" "./morello/morello.md" "./mve_intrinsics/mve.md" "./neon_intrinsics/advsimd.md"; do
+  echo "Checking $file..."
+
+  # The following string of pipes extracts the unresolved internal links by:
+  # - Using tee to output the error and warning messages generated by pandoc pdf conversion into a file
+  # - Using grep to extract all unresolved links from the file, eg "pdfTeX warning (dest): name{ssec-bf16-scalar}"
+  # - Using wc to count all the lines and thus all the warnings/detected broken links
+  pandoc $file --verbose -o pdfs/tmp.pdf 2>&1 | tee output
```

Comment:
The `pdfs` folder is not yet created here. I see this message in the CI: `pandoc: pdfs/tmp.pdf: openBinaryFile: does not exist (No such file or directory)`

I think you want to output to `/dev/null`, forcing pandoc to use PDF output with `--to=pdf`:

```suggestion
  pandoc $file --verbose --to=pdf -o /dev/null 2>&1 | tee output
```

I also suggest that you artificially break on purpose one of the internal references to make sure the script is detecting the failures

Reply:
The command above doesn't seem to be working on either my remote run or the one on github - might I suggest we output the tmp file in the root directory instead?

Reply:
> might I suggest we output the tmp file in the root directory instead?
I am surprised, but sgtm. That, or just `mkdir -p pdfs` before invoking pandoc